README.md

```diff
@@ -12,6 +12,15 @@ Setting up
 4. Go to `localhost:3000/users/<vine-user-id>/atom` or 
    `localhost:3000/users/<vine-user-id>/rss`
 
+Usage with systemd
+------------------
+
+1. Copy `vinefeed.service` and `vinefeed.socket` to a systemd units directory, e.g. `/usr/lib/systemd/system` or `~/.config/systemd/user`
```

Comment:
Maybe "Copy or symlink", if symlinks would work too? I realize it's a bad idea to symlink if it's going to be root running it though.